content/en/security/application_security/threats/add-user-info.md

```diff
@@ -714,29 +714,59 @@ Once saved, the rule is deployed to instances of the service that have Remote Co
 
 ## Automatic user activity event tracking
 
-When ASM is enabled, recent Datadog Tracing Libraries attempt to detect user activity events automatically.
+When ASM is enabled, Datadog Tracing Libraries attempt to detect user activity events automatically.
 
 The events that can be automatically detected are:
 
 - `users.login.success`
 - `users.login.failure`
 - `users.signup`
 
-### Automatic user activity event tracking mode
+### Automatic user activity event tracking modes
 
-Automatic user activity tracking offers two modes: <code>safe</code>, and <code>extended</code>
+Automatic user activity tracking offers the following modes:
 
-In <code>safe</code> mode, the trace library does not include any PII information on the events metadata. The tracer library tries to collect the user ID, and only if the user ID is a valid [GUID][10]
+- `identification` mode (short-name: `ident`): 
+  - By default, ASM libraries automatically collect user IDs.
+  - This mode is the default and always collects the user ID or best effort.
+  - The user ID is collected on login success/failure. With failure, the user ID is collected regardless of whether the user exists or not.
+  - When the instrumented framework doesn’t clearly provide a user ID, but rather a structured user object, the user ID is determined on a best effort basis based on the object field names. This list of field names are considered, ordered by priority:
+    - `id`
+    - `email`
+    - `username`
+    - `login`
+    - `user`
+  - If no user ID is available or found, the user event is not emitted.
+- `anonymization` mode (short-name: `anon`):
+  - This mode is the same as `identification`, but anonymizes the user ID.
+- `disabled` mode:
+  - ASM libraries do *not* collect any user ID from their automated instrumentations. 
+  - This property also results in no longer emitting user login events.
 
-In <code>extended</code> mode, the trace library tries to collect the user ID, and the user email. In this mode, we do not check the type for the user ID to be a GUID. The trace library reports whatever value can be extracted from the event.
+<div class="alert alert-info">All modes only affect automated instrumentation. The modes don't apply to manual collection. Manual collection is confiigured using an SDK, and those settings are not overriden by automated instrumentation.</div>
 
-To configure automatic user event tracking mode, you can set the environment variable <code>DD_APPSEC_AUTOMATED_USER_EVENTS_TRACKING</code> to <code>safe</code> or <code>extended</code>. By default, the tracer library uses the <code>safe</code> mode.
+Datadog libraries allow you to configure auto-instrumentation by using the `DD_APPSEC_AUTO_USER_INSTRUMENTATION_MODE` environment variable with the short name for the mode: `ident`|`anon`|`disabled`.
+
+The default mode is `identification` mode (short-name: `ident`).
```

Comment:
I suggest "short name" instead of "short-name" for all of these instances (there are some above this one too). I would expect the noun to just be "short name," as it appears in the paragraph above this one.